src/open_inwoner/ssd/templates/jaaropgave.html

```diff
@@ -2,6 +2,9 @@
 
 <!DOCTYPE html>
 <html lang="nl">
+<head>
+  <title></title>
+</head>
```

Comment:
Why was this added?

Reply:
The title is duplicated in case a new tab is opened for file download (at least on Firefox). Apparently it is not (yet) standardized what browsers do with the title. The `<title><title>` effectively re-sets whatever the browser does with the title, afterwards we define our custom title. It's a hack, but I don't know a better way to deal with this. Perhaps @jiromaykin has some idea about this. I did however remove the extra `head`, which was an oversight.

Reply:
Discussed with Jiro, I'll revert changes and create a separate issue for this because it's an accessibility issue.